herbert_core/utilities/classes/@object_lookup/object_repmat.m

```diff
@@ -0,0 +1,177 @@
+function obj_out = object_repmat (obj, varargin)
```

Comment:
Some changes now made. The operation of object_repmat does more than a repmat on a single object - it can perform an *implicit* repmat on a subset of a collection of objects. However, I agree that the '-repeat' keyword syntax is an unnecessary confusion and the keyword itself can be removed to leave the syntax closer to that of of a vanilla repmat. Object_repmat does need the indices of the subset of stored object arrays as the second argument (after the object_lookup object itself), but '-repmat' is truly unnecessary.  I left it from an earlier version so as not to require changing a couple of Tobyfit routines.